Comp2022/src/main/cpp/RobotContainer.cpp

```diff
@@ -96,7 +97,8 @@ RobotContainer::RobotContainer() :
 
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DEFAULT-COMMANDS
 
-    m_climber.SetDefaultCommand(ClimberRun(&m_climber));
+    m_climber.SetDefaultCommand(ClimberMoveHeight(Climber::NOCHANGE_HEIGHT, &m_climber));

```

Comment:
Wait so the default command just leaves the climber in its current position, right? Out of curiosity, is there a reason the default isn't stowing it?

Reply:
Default cmd isn't going to the stow position because depending on other states, the climber heights will be different, so we're not binding it to go to 0.25 inches every time a grouped cmd ends, but it will just hold to the height it's currently at

The way I understand it/ an example is that currently we have a button to toggle on and off ClimberRun, but every time we press it to end the cmd, we don't want the climber to move back to the stow position - hopefully this makes sense!